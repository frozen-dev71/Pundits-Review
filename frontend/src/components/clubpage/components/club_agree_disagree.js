// ======================================
// CLUBS - Agree / Disagree Button Component allowing users to interact with club pages depending on their opinion
// Will post to the DB with number of agrees / disagrees
// ======================================


// ===========
// IMPORTS 
// ===========

// General Imports 
import React, { Component } from "react";
import { render } from "react-dom";
import axios from 'axios';
import { connect } from "react-redux";


// Local JS Imports 
import { updateClubsAgreeDisagreeScore, setClubsAgreeDisagreeScore, locallyUpdateAgreeDisagreeScoreClubs} from "../../actions/actions";
import { AgreeDisagreeScore } from '../../playerpage/components/agree_disagree_score';

// CSS Imports 
import '../../playerpage/components/agree_disagree.css';

// ===========
// Agree Disagree Component
// ===========

export class ClubAgreeDisagreeContainer extends Component {

	constructor(props) {
		super(props);
		this.state = {

			buttonDisabled: false
		}

		this._handleClickAgree = this._handleClickAgree.bind(this);
		this._handleClickDisagree = this._handleClickDisagree.bind(this);
	}

	componentDidMount() {


		this.props.setClubsAgreeDisagreeScore(this.props.clubName)

	}

	_handleClickAgree(e) {

		let currentClub = this.props.clubsViewed[this.props.clubsViewed.length - 1];

		updateClubsAgreeDisagreeScore(currentClub, true);

		let newScoreObject = this.props.clubAgreeDisagreeScores[this.props.clubAgreeDisagreeScores.length -1];

		newScoreObject.agree_score += 1;

		this.props.locallyUpdateAgreeDisagreeScoreClubs(newScoreObject);

		// Disable the button
		this.setState({
			buttonDisabled: true
		})

	}

	_handleClickDisagree(e) {

		let currentClub = this.props.clubsViewed[this.props.clubsViewed.length - 1];

		updateClubsAgreeDisagreeScore(currentClub, false);

		let newScoreObject = this.props.clubAgreeDisagreeScores[this.props.clubAgreeDisagreeScores.length -1];

		newScoreObject.disagree_score += 1;

		this.props.locallyUpdateAgreeDisagreeScoreClubs(newScoreObject);

		// Disable the button
		this.setState({
			buttonDisabled: true
		})

	}

	render() {

		let result = (

			<div className="agree-disagree-wrapper">

				<div className="agree-disagree-button-container">

					<p className="agree-disagree-heading-text body-text-light">AGREE</p>
					<button id="agree-button" className="agree-disagree-button" disabled={this.state.buttonDisabled} onClick={(e) => this._handleClickAgree(e)}><span className="agree-disagree-icon fa fa-check"></span></button>
					<AgreeDisagreeScore choice={"agree_score"} clubOrPlayer={"club"}/>

				</div>

				<div className="agree-disagree-button-container">

					<p className="agree-disagree-heading-text body-text-light">DISAGREE</p>
					<button id="disagree-button" className="agree-disagree-button" disabled={this.state.buttonDisabled} onClick={(e) => this._handleClickDisagree(e)}><span className="agree-disagree-icon fa fa-times"></span></button>
					<AgreeDisagreeScore choice={"disagree_score"} clubOrPlayer={"club"}/>

				</div>

			</div>

			);

		return result;
	}
}


// ===============
// Connect redux state container to PlayerPage component
// ===============


const mapStateToProps = state => {
  return { 
  		clubsViewed: state.clubsViewed,
	  	clubAgreeDisagreeScores: state.clubAgreeDisagreeScores
  	};
};


export const ClubAgreeDisagree = connect(mapStateToProps, { setClubsAgreeDisagreeScore, locallyUpdateAgreeDisagreeScoreClubs })(ClubAgreeDisagreeContainer);

