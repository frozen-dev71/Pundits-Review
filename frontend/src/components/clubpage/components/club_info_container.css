/*
// ======================================
// CSS for the Player Info container on the CLUB PAGE
// ======================================
*/


.club-info-container-wrapper {

	display: grid;
    grid-template-columns: repeat(6,1fr);
    grid-template-rows: repeat(2,1fr);
    height: 20vh;
    padding: 1em 0;
    align-items: center;
    margin-top: 10vh; 
    grid-gap: 1em;
    width: 75vw;
}

.club-page-club-info-name  {
	grid-row: 1;
	grid-column: 3/7;
	font-size: 1.25em;
}

.club-badge-club-page-info-container {
	width: 7em;
	height: 7em;
	grid-row: 1/3;
	grid-column: 1/3;
}

.club-page-club-info-stadium {
	grid-row: 2;
    grid-column: 3/7;
    align-self: flex-start;
    margin-left: 1em;
}


/*
// ======================================
// MEDIA QUERIES
// ======================================
*/


@media (min-width: 600px) {

    .club-info-container-wrapper {
        width: 50vw;
    }

    .club-page-club-info-name  {
        font-size: 1.5em;
    }

    .club-badge-club-page-info-container {
        width: 9em;
        height: 9em;
    }
}

@media (min-width: 800px) {


    .club-badge-club-page-info-container {
        width: 11em;
        height: 11em;
    }
}




@media (min-width: 900px) {

    .club-info-container-wrapper {
        width: 40vw;
    }
}

@media (min-width: 1000px) {

    .club-page-club-info-name  {
        font-size: 2em;
    }

    .club-page-club-info-stadium {
        font-size: 1.25em;
    }
}



@media (min-width: 1200px) {

    .club-info-container-wrapper {
        width: 30vw;
    }
}