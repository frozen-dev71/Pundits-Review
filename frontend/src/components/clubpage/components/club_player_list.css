/*
// ======================================
// CSS for the Club Player List Component
// ======================================
*/

.club-player-list-wrapper {
	display: flex;
	flex-direction: column;
	align-items: center;
	width: 95vw;
	padding: 1em 0;

}

.club-player-list-title {
	font-size: 1.25em;
	margin: 2em 0;
}



/*
// ======================================
// MEDIA QUERIES
// ======================================
*/

@media (min-width: 800px) {

	.club-player-list-title {
		font-size: 1.5em;
		margin: 3em 0;
	}

}



@media (min-width: 1000px) {

	.club-player-list-title {
		font-size: 1.75em;

	}

}