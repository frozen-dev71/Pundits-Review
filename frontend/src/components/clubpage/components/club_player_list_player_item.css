/*
// ======================================
// CSS for the Individual Player Item Container - used within Club Player List
// ======================================
*/

.club-player-list-player-item-container {
	
	display: flex;
	justify-content: space-around;
	width: 80vw;
    padding: 1em 0;
    align-items: center;
}

.club-player-list-player-item-name {
	width: 80%;
	font-size: 1.25em;
	color: black;
}


/*
// ======================================
// MEDIA QUERIES
// ======================================
*/

@media (min-width: 650px) {

	.club-player-list-player-item-container {

		width: 50vw;
	}
}


@media (min-width: 1000px) {

	.club-player-list-player-item-container {

		padding: 2em 0;
	}

	
	.club-player-list-player-item-name {
		width: 80%;
		font-size: 1.5em;
		color: black;
	}

}



@media (min-width: 1200px) {

	.club-player-list-player-item-container {

		width: 40vw;
	}
}