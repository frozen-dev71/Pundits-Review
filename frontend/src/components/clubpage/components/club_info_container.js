// ======================================
// Club Info Container Component for use within the Club page - club_name, image & stadium
// ======================================



// ===========
// IMPORTS 
// ===========

// General Imports 
import React, { Component } from "react";
import { render } from "react-dom";

// Local JS Imports
import { badgeFinder } from "../../global/functions/set_data_helpers";
import { stadiumNames } from '../../global/resources/club_stadiums';

// CSS Import
import './club_info_container.css';


// ===========
// Club Info Component 
// ===========



export class ClubInfoContainer extends Component {

	render() {

		let result = (

				<div className="club-info-container-wrapper">

					<p className="club-page-club-info-name heading-text" >{this.props.clubData.club}</p>
					<span className={this.props.clubBadgeClassName} ></span>
					<p className="club-page-club-info-stadium body-text-light" >{stadiumNames[this.props.clubData.club]}</p>

				</div>

			);


		return result;
	}
}