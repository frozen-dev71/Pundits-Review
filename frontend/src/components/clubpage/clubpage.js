// ======================================
// This file contains the top level CLUB PAGE component that will wrap other elements on the CLUB PAGE 
// All other CLUB PAGE components will be directed through this component
// ======================================


// ===========
// IMPORTS 
// ===========

// General Imports 
import React, { Component } from "react";
import { render } from "react-dom";
import { connect } from "react-redux";
import { Helmet } from 'react-helmet';

// Local JS Imports 
import { ClubInfoContainer } from './components/club_info_container';
import { setReviewScore, setReviewSentences, badgeFinder } from "./../global/functions/set_data_helpers";
import { getCrawlDates } from './../actions/actions';  
import { DividingLine } from  "./../global/components/dividing_line";
import { InfoBar } from './../playerpage/components/infobar';
import { ReviewScoreContainer } from './../playerpage/components/review_score_container';
import { SeasonPerformanceContainer } from './../playerpage/components/season_performance_container';
import { SnippetsContainer } from './../playerpage/components/snippets_container';
import { ClubAgreeDisagree } from './components/club_agree_disagree';
import { ClubPlayerList } from './components/club_player_list';
import { EmptyPlayerClubPage } from './../global/components/empty_player_club_page';
import { Footer } from './../global/components/site_footer/footer';

// Local CSS Imports
import './../playerpage/playerpage.css';



// ===========
// Club Page Component
// ===========


export class ClubPageWrapper extends Component {

	constructor(props) {
    super(props);
    this.state = {
    		clubData: [],
    		reviewScore: -100,
    		positiveReviews: 0,
    		negativeReviews: 0,
    		totalReviews: 0,
    		snippets: [],
    		clubBadgeClassName: "",
    		agreeScore: 1, 
    		disagreeScore: 1,
    		loading: true
    	};

    this.componentDidUpdate = this.componentDidUpdate.bind(this);

  	}


  	componentDidMount() {

  		this.componentDidUpdate();

  	}

  	componentDidUpdate() {


  		// Set the new CLUB

		if (this.props.clubsViewed.length && this.props.clubsViewed[this.props.clubsViewed.length - 1] !== this.state.clubData) {

			this.props.getCrawlDates();

			this.setState({ 
				clubData: this.props.clubsViewed[this.props.clubsViewed.length - 1],
				reviewScore: setReviewScore(this.props.clubsViewed[this.props.clubsViewed.length - 1], true)[0],
				positiveReviews: setReviewScore(this.props.clubsViewed[this.props.clubsViewed.length - 1], true)[1],
				negativeReviews: setReviewScore(this.props.clubsViewed[this.props.clubsViewed.length - 1], true)[2],
				totalReviews: setReviewScore(this.props.clubsViewed[this.props.clubsViewed.length - 1], true)[3],
				snippets: setReviewSentences(this.props.clubsViewed[this.props.clubsViewed.length - 1], true),
				clubBadgeClassName: badgeFinder(this.props.clubsViewed[this.props.clubsViewed.length - 1].club),
				loading: false
			});

		}
  	}


	render() {

		let result = <EmptyPlayerClubPage playerOrClub={"club"} />

		if (this.state.loading === false) {

			result = (

				<div className="player-club-page-wrapper">

					<Helmet>
				
						<title>{this.state.clubData.club + " - Pundits Review"}</title>
						<meta property="og:description" content={this.state.clubData.club + "'s Pundits Review"}/>
						<meta property="og:url" content={"https://www.punditsreview.com/" + encodeURIComponent(this.state.clubData.club) }/>
			    		<meta property="og:type" content="football blog data"/>

			    	</Helmet>

					<InfoBar clubSpecified={true} clubData={this.state.clubData} playerClubBadgeClassName={"infobar-badge " + this.state.clubBadgeClassName}/>
					
					<ClubInfoContainer id="club-info-container" clubData={this.state.clubData} clubBadgeClassName={"club-badge-club-page-info-container " + this.state.clubBadgeClassName}/>

					<DividingLine />

					<ReviewScoreContainer reviewScore={this.state.reviewScore} positiveReviews={this.state.positiveReviews} negativeReviews={this.state.negativeReviews} totalReviews={this.state.totalReviews} />

					<ClubAgreeDisagree clubName={this.state.clubData.club}/>

					<DividingLine />

					<SeasonPerformanceContainer data={this.state.clubData} keyCode={"club_scores"}/>

					<DividingLine />

					<p id="snippets-title" className="snippets-title heading-text">Snippets</p>

					<SnippetsContainer snippets={this.state.snippets}/>

					<DividingLine />

					<ClubPlayerList clubName={this.state.clubData.club} players={this.state.clubData.players}/>

					<Footer />

				</div>

			);


		}
		


		return result;
	}
}




// ===============
// Connect redux state container to PlayerPage component
// ===============


const mapStateToProps = state => {
  return { 
	  	clubsViewed: state.clubsViewed
  	};
};


export const ClubPage = connect(mapStateToProps, { getCrawlDates })(ClubPageWrapper);

