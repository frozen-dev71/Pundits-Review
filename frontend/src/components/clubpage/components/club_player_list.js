// ======================================
// Player List Component - Displays list of players & review scores belonging to current club
// ======================================



// ===========
// IMPORTS 
// ===========

// General Imports 
import React, { Component } from "react";
import { render } from "react-dom";

// Local JS Imports
import { ClubPlayerListPlayerItem } from './club_player_list_player_item';
import { setReviewScore, returnDateMatchArray } from '../../global/functions/set_data_helpers';

// CSS Imports 
import './club_player_list.css';

// ===========
// Club Player List Component
// ===========


export class ClubPlayerList extends Component {

	render() {

		let result = <span></span>;

		if (this.props.players) {

			result = (

				<div className="club-player-list-wrapper">

					<p className="club-player-list-title heading-text">{this.props.clubName + " Players"}</p>

					{this.props.players.map((player, index) => {

						let dateArray = returnDateMatchArray(player.scores);

						if (dateArray.length) {

							let playerReviewScore = dateArray[0].sentiment_score;
							let playerPositive = dateArray[0].n_positive;
							let playerNegative = dateArray[0].n_negative;

							if ((playerPositive + playerNegative) > 3) {

								return <ClubPlayerListPlayerItem key={index} playerName={player.player} reviewScore={Math.round(playerReviewScore * 100)} />

							}

						}

					})}


				</div>


			);


		}


		return result;
	}
}