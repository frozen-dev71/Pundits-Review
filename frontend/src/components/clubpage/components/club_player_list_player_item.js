// ======================================
// Club Player List Individual Player Item Component - displays player name & review score
// ======================================


// ===========
// IMPORTS 
// ===========

// General Imports 
import React, { Component } from "react";
import { render } from "react-dom";
import { Link } from 'react-router-dom';
import { connect } from "react-redux";

// Local JS Imports 
import { ReviewScoreCircle } from '../../results/components/review_score_circle';
import { setPlayer } from '../../actions/actions';

// CSS Imports 
import './club_player_list_player_item.css';

// ===========
// Club Player List Player Item Component
// ===========


export class ClubPlayerListPlayerItemWrapper extends Component {

	_handleClick(playerName) {

		this.props.setPlayer(this.props.playerName);

	}

	render() {

		let playerNameParam = "/players/" + encodeURIComponent(this.props.playerName);

		let result = (

				<Link to={playerNameParam} className="club-player-list-player-item-container react-link" onClick={() => this._handleClick(this.props.playerName)}>

					<p className="club-player-list-player-item-name body-text-light">{this.props.playerName}</p>

					<ReviewScoreCircle reviewScore={this.props.reviewScore} />

				</Link>

			);


		return result;
	}
}


// ===============
// Connect redux state container to ClubPlayerListPlayerItem component
// ===============


const mapStateToProps = state => {
  return { 

	  	playersViewed: state.playersViewed
  	};
};



// Export connected component
export const ClubPlayerListPlayerItem = connect(mapStateToProps, { setPlayer })(ClubPlayerListPlayerItemWrapper);