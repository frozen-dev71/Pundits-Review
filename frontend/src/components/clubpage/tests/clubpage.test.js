// ======================================
// This file tests for the components listed in the CLUBPAGE sub-directory
// ======================================


// ===========
// TEST COVERAGE
// ===========

// 1. Check that ClubPage renders correctly
// 2. Check that ClubAgreeDisagree renders correctly
// 3. Check that ClubInfoContainer renders correctly
// 4. Check that ClubPlayerList renders correctly
// 5. Check that ClubPlayerListPlayerItem renders correctly

// ===========
// IMPORTS 
// ===========


// General Imports
import React from 'react';
import Enzyme, { shallow, mount, render } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import { Provider } from "react-redux";
import toJson from 'enzyme-to-json';
import { BrowserRouter } from 'react-router-dom';

// Local JS Imports
import { ClubPage } from '.././clubpage';
import { ClubAgreeDisagree } from '.././components/club_agree_disagree';
import { ClubInfoContainer } from '.././components/club_info_container';
import { ClubPlayerList } from '.././components/club_player_list';
import { ClubPlayerListPlayerItem } from '.././components/club_player_list_player_item';

import store from '../../store/store';

// Configurations
Enzyme.configure({ adapter: new Adapter() })

// ===========
// Suppression of SSL Warning 
//
// Code adapted from comment from sketchbuch on GitHub issues post 'Warning: useLayoutEffect does nothing on the server #1373' on 12 Oct 2019
// https://github.com/reduxjs/react-redux/issues/1373
// Accessed 09 Sep 2020
//
// __Note__:
//	
//		- Jest produces "Warning: useLayoutEffect does nothing on the server" & 'Warning: React.createElement: type is invalid -- expected a string (for built-in components)' - warning which confuses test cases 
// 		- Wrapper test suppresses that warning for clear test reports
// ===========

describe('ClubPage Tests', () => {
  	const originalConsoleError = console.error;

  	beforeEach(() => {
    	console.error = jest.fn((msg) => {
	      	if (msg.includes('Warning: useLayoutEffect does nothing on the server') || msg.includes('Warning: React.createElement: type is invalid -- expected a string (for built-in components)')) {
	       		return null;
	      	} else {
	        	originalConsoleError(msg);
	      	}
    	});
  	});

  afterEach(() => {
    console.error = originalConsoleError;
  });


// End of referenced code

  	// ===========
	// TEST 1. Check that ClubPage renders correctly
	// ===========

	describe('ClubPage', () => {

	  	it('should render correctly', () => {

		  	// Declare component
		  	const component = (<BrowserRouter><Provider store={store}><ClubPage/></Provider></BrowserRouter>);
			
			//
			// Simulate Component
		    const wrapper = render(component);

		    // Run Test Case
		    expect(toJson(wrapper)).toMatchSnapshot();

	  	});

	})


	// ===========
	// TEST 2. Check that ClubAgreeDisagree renders correctly
	// ===========

	describe('ClubAgreeDisagree', () => {

	  	it('should render correctly', () => {

		  	// Declare component
		  	const component = (<BrowserRouter><Provider store={store}><ClubAgreeDisagree/></Provider></BrowserRouter>);
			
			//
			// Simulate Component
		    const wrapper = render(component);

		    // Run Test Case
		    expect(toJson(wrapper)).toMatchSnapshot();

	  	});

	})


	// ===========
	// TEST 3. Check that ClubInfoContainer renders correctly
	// ===========

	describe('ClubInfoContainer', () => {

	  	it('should render correctly', () => {

		  	// Declare component
		  	const component = (<BrowserRouter><Provider store={store}><ClubInfoContainer/></Provider></BrowserRouter>);
			
			//
			// Simulate Component
		    const wrapper = shallow(component);

		    // Run Test Case
		    expect(toJson(wrapper)).toMatchSnapshot();

	  	});

	})


	// ===========
	// TEST 4. Check that ClubPlayerList renders correctly
	// ===========

	describe('ClubPlayerList', () => {

	  	it('should render correctly', () => {

		  	// Declare component
		  	const component = (<BrowserRouter><Provider store={store}><ClubPlayerList/></Provider></BrowserRouter>);
			
			//
			// Simulate Component
		    const wrapper = render(component);

		    // Run Test Case
		    expect(toJson(wrapper)).toMatchSnapshot();

	  	});

	})


	// ===========
	// TEST 5. Check that ClubPlayerListPlayerItem renders correctly
	// ===========

	describe('ClubPlayerListPlayerItem', () => {

	  	it('should render correctly', () => {

		  	// Declare component
		  	const component = (<BrowserRouter><Provider store={store}><ClubPlayerListPlayerItem/></Provider></BrowserRouter>);
			
			//
			// Simulate Component
		    const wrapper = render(component);

		    // Run Test Case
		    expect(toJson(wrapper)).toMatchSnapshot();

	  	});

	})



});
