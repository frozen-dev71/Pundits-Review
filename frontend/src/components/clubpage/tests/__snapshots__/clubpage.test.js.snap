// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ClubPage Tests ClubAgreeDisagree should render correctly 1`] = `
<div
  class="agree-disagree-wrapper"
>
  <div
    class="agree-disagree-button-container"
  >
    <p
      class="agree-disagree-heading-text body-text-light"
    >
      AGREE
    </p>
    <button
      class="agree-disagree-button"
      id="agree-button"
    >
      <span
        class="agree-disagree-icon fa fa-check"
      />
    </button>
    <span />
  </div>
  <div
    class="agree-disagree-button-container"
  >
    <p
      class="agree-disagree-heading-text body-text-light"
    >
      DISAGREE
    </p>
    <button
      class="agree-disagree-button"
      id="disagree-button"
    >
      <span
        class="agree-disagree-icon fa fa-times"
      />
    </button>
    <span />
  </div>
</div>
`;

exports[`ClubPage Tests ClubInfoContainer should render correctly 1`] = `
<Router
  history={
    Object {
      "action": "POP",
      "block": [Function],
      "createHref": [Function],
      "go": [Function],
      "goBack": [Function],
      "goForward": [Function],
      "length": 1,
      "listen": [Function],
      "location": Object {
        "hash": "",
        "pathname": "/",
        "search": "",
        "state": undefined,
      },
      "push": [Function],
      "replace": [Function],
    }
  }
>
  <Provider
    store={
      Object {
        "dispatch": [Function],
        "getState": [Function],
        "replaceReducer": [Function],
        "subscribe": [Function],
        Symbol(observable): [Function],
      }
    }
  >
    <ClubInfoContainer />
  </Provider>
</Router>
`;

exports[`ClubPage Tests ClubPage should render correctly 1`] = `
<div
  class="empty-page-wrapper"
>
  <p
    class="no-results-text body-text-light"
  >
    It looks like you haven't selected a club 
  </p>
  <div
    class="dividing-line background-dark"
  />
  <p
    class="no-results-sub-text body-text-light"
  >
    Head back to the homepage to make a new search for a club
  </p>
  <a
    class="react-link empty-page-home-link body-text-light"
    href="/"
  >
    HOME
  </a>
</div>
`;

exports[`ClubPage Tests ClubPlayerList should render correctly 1`] = `<span />`;

exports[`ClubPage Tests ClubPlayerListPlayerItem should render correctly 1`] = `
<a
  class="club-player-list-player-item-container react-link"
  href="/players/undefined"
>
  <p
    class="club-player-list-player-item-name body-text-light"
  />
  <div
    class="review-score-circle body-text-light white-text background-light"
  />
</a>
`;
